src/gpu/intel/ocl/micro_sdpa.cpp

```diff
@@ -325,8 +325,9 @@ status_t micro_sdpa_t::pd_t::init_microkernels(impl::engine_t *engine) {
         problem_vs.aOffset = ABOffset::Calc;
     }
     if (with_value_scales() || with_value_zp()) {
-        problem_vs.aqGroupM
-                = (vs_common_scales || vs_common_zp) ? 1 : value_group_size();
+        problem_vs.aqGroupM = (vs_common_scales || vs_common_zp)
+                ? 1
+                : utils::rnd_up_pow2(value_group_size());
```

Comment:
This is only safe if there is only one group in the `m` dimension (if group size >= head size).

Reply:
Ah good point. I will add a check for this.

Reply:
Added the check here: https://github.com/oneapi-src/oneDNN/pull/2352/files#diff-bba4b0f9326dd28e41aa9188d531360420f3040755ba76beb260df3441365641R144